clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -913,7 +913,8 @@ private ConsumerCoordinator buildCoordinator(Metrics metrics, List<PartitionAssi
                 retryBackoffMs,
                 defaultOffsetCommitCallback,
                 autoCommitEnabled,
-                autoCommitIntervalMs);
+                autoCommitIntervalMs,
+                null);
```

Comment:
I already added ConsumerInterceptors unit test, and this new test would just make sure that ConsumerCoordinator calls ConsumerInterceptors methods in the right places. And this is already tested in the integration tests. So this one would be an intermediate test, which would not really add anything new to unit + integration tests that I already added.